src/api/roles/client/fitness.py

```
@@ -367,21 +520,33 @@ def delete_client_workout_plan(
+        message=f"{acc.name} cancelled '{plan_name}'",
+        details=f"Was scheduled {_fmt_block(cwp.start_time, cwp.end_time)}",
+    )
+
     db.commit()
     return {"details": "deleted"}
 
 @router.get("/daily-survey/today", response_model=DailySurveyResponse)
 def get_today_daily_survey(
+    local_date: Optional[str] = None,
+    tz_offset_minutes: Optional[int] = None,
     db: Session = Depends(get_session),
```

Ruff (0.15.12)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)